tests/test_http.py

```diff
@@ -215,3 +216,18 @@ def test_set_timeout(self):
             .once())
 
         HttpStream(url, method, verify_ssl=False, **kwargs)
+
+
+class TestHttpResponse(object):
+    def test_simple_response(self):
+        content = 'this is content'
```

Comment:
I think the content should be a 'bytes' instance, e.g. `content = b'this is content'`. This is the difference between requests.Response.content and requests.Response.text -- 'text' has been decoded according to the media type header. (Note that for json we have to decode 'content' separately using guess_json_utf() because the media type header application/json gives no clue about encoding.)

Reply:
This is likely the reason why tests failed in python 3.